src/Utils/TypeInfo.php

```diff
@@ -339,11 +338,12 @@ static function ($arg) use ($node) {
                 }
                 $this->argument            = $argDef;
                 $this->defaultValueStack[] = $argDef && $argDef->defaultValueExists() ? $argDef->defaultValue : Utils::undefined();
-                $this->inputTypeStack[]    = Type::isInputType($argType) ? $argType : null;
+                $this->inputTypeStack[]    = $argType;
```

Comment:
I think this is wrong. The issue is that `isInputType` additionally checks that the whole chain of possibly wrapped types are also input types. Without this check `[ObjectType]` will qualify as an `InputType` (because `ListOf` wrapper is both Output and Input type). We are probably lacking some tests for this.

Reply:
Sounds good. Restored in 3b428f7